src/azure-cli-core/azure/cli/core/__init__.py

```diff
@@ -4,7 +4,7 @@
 # --------------------------------------------------------------------------------------------
 # pylint: disable=line-too-long
 
-__version__ = "2.34.0"
+__version__ = "2.34.1"
```

Comment:
Please do not upgrade CLI version manually. It will be automatically generated by CI pipeline when releasing